addons/account/models/account_move.py

```diff
@@ -1205,8 +1205,8 @@ def _compute_tax_totals(self):
                 move.tax_totals = self.env['account.tax']._prepare_tax_totals(**kwargs)
                 rounding_line = move.line_ids.filtered(lambda l: l.display_type == 'rounding')
                 if rounding_line:
-                    amount_total_rounded = move.tax_totals['amount_total'] - rounding_line.balance
-                    move.tax_totals['formatted_amount_total_rounded'] = formatLang(self.env, amount_total_rounded, currency_obj=move.currency_id) or ''
+                    move.tax_totals['amount_total_rounded'] = move.amount_total
```

Comment:
This doesn't work in the form view without saving because `amount_total` depends on debit/credit, but these values are not always correctly computed before saving the record

Reply:
Thanks for the review, I did not realize it was the cause of why I had to save to see the proper amount